sklearn/inspection/_plot/partial_dependence.py

```diff
@@ -465,7 +530,22 @@ def plot(self, ax=None, n_cols=3, line_kw=None, contour_kw=None):
         default_contour_kws = {"alpha": 0.75}
         contour_kw = {**default_contour_kws, **contour_kw}
 
+        default_line_kws = {'color': 'C0'}
+        line_kw = {**default_line_kws, **line_kw}
+        individual_line_kw = line_kw.copy()
+
+        if self.individual == 'both':
+            individual_line_kw['alpha'] = 0.3
```

Comment:
I think we should use `alpha < 1` all the time. High density regions (overlapping lines) are easier to spot with lower opacity.

Reply:
Fair enough. Implemented in https://github.com/scikit-learn/scikit-learn/pull/16619/commits/4e0337d13449834d42c4fa07468763e56b87abd3